src/adapters/superagent.js

```diff
@@ -1,30 +1,13 @@
 import superagent from 'superagent';
 import * as httpMethods from '../constants/http-methods';
 
-export const createRequest = (url, method) => {
```

Comment:
can you keep this function and the switch statement for explicitness?

in addition, we can move to making this function not exported with your change in tests/. that would be great